CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+   the default strategy of `--no-ff` merges ([configured][merge-methods-github] in GitHub), you can press the merge
+   button or issue a local `git flow ... finish` without *ever* having to force push. 
+
+#### Forking
+While core contributors will work directly on the original repo, outside contributors will need to maintain their own 
+fork of Lookit while working on new features for (or fixing bugs for) the platform. We recommend reading this handy
+[guide on forking][forking-workflow-walkthrough] if you are not familiar with this workflow. GitHub's own [`hub` 
+utility][hub-oss-contrib] will make your life a lot easier in this regard by abstracting over some of the steps; you 
+can find the installation instructions [here][hub-install] (or, if you like to live on the cutting-edge, try out 
+GitHub's [shiny new CLI][github-cli]).
+
+#### Staying on the Command Line
+If you have `hub` and `gitflow-avh` installed, you should be able to work entirely from the command line - even when
+working with a PR of a published feature/bugfix branch.
+
+You'll need to change a bit of `gitflow-avh`'s default `finish` behavior to push to remote branches:
+```bash
+git config --local gitflow.bugfix.finish.push yes
+git config --local gitflow.feature.finish.push yes
+git config --local gitflow.release.finish.push yes
+```
+This will enact proper Git Flow "finalizer" processes on remote branches (e.g. merging `release` branches back to 
+`develop` and `master`, deleting old `feature` and `bugfix` branches). This way, you won't have to do manual (remote) 
+branch cleanup with vanilla git commands, or have to remember to push tags.
+
+Once that's done, you'll just do normal Git Flow process with one exception: after running a `git flow ... publish` 
+command, you'll run `hub pull-request` and fill out a proper PR with your command line editor of choice.
+
+Since GitHub considers a PR "Merged" as soon as the commits in the PR branch are also found in the target branch (so
+long as the commits are not modified in any way - not a problem if you never rebase or amend remote commits!), you can 
+just issue a `git flow ... finish` command to do the necessary merges, tag pushes, and branch cleanup.
+
+### Localization
+At the moment, we have very rudimentary i18n and l10n processes - nothing like the sophisticated pipeline you might see
+at a big company. It's a system based on trust:
+
+1. Code contributors and reviewers are trusted to enforce internationalization of strings where appropriate.
+2. Translators are expected to keep their `feature` branches up-to-date with `develop`. 
+3. Translators should keep their `.po` files up to date by running `./manage.py makemessages` whenever `develop` is 
+   merged back into their `feature` branch. 
+3. Translators will be notified ahead of time when releases are about to cut so that they can get in any last-minute 
+   translations.
```

Comment:
it feels like we probably need more of a step-by-step guide for translators here (does each one have a feature branch for their own language, up-to-date with develop?) Also do we definitely want them to be the ones running makemessages or do we want to do that for them once, they each get the changes and add their edits?

(BUT that guide might also belong in the docs, I'm not sure)